adafruit_lis3dh.py

```diff
@@ -442,7 +471,15 @@ class LIS3DH_SPI(LIS3DH):
 
     """
 
-    def __init__(self, spi, cs, *, baudrate=100000, int1=None, int2=None):
+    def __init__(
+        self,
+        spi: SPI,
+        cs: digitalio.DigitalInOut,
+        *,
+        baudrate: Optional[int] = 100000,
```

Comment:
This is another one that should just be `int`.

Reply:
Tks see https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/pull/76/commits/59c669238f330e5703bb89641199c02303db38c7